simulators/active_surface.py

```diff
@@ -1,9 +1,8 @@
 #!/usr/bin/python
+import time
```

Comment:
This changes are not related to this pull request.  Please open another issue pointing out the problem and fix it in another branch.